compose/backend.yml

```diff
@@ -43,6 +43,7 @@ services:
       ELASTIC_SEARCH_ENABLED: "${ES_ENABLED_DOCKER}"
       ELASTIC_SEARCH_HOST: "ccd-elasticsearch"
       ELASTIC_SEARCH_FAIL_ON_IMPORT: "true"
+      DEFINITION_STORE_TX_TIMEOUT_DEFAULT: "${DEF_STORE_TIMEOUT:-30}"
```

Comment:
maybe increase default to 60, if it start timouting, noone (excpet you :P ) will know that it needs to be changed here

Reply:
30 in the app, but can increase to 60 by default